integration/benchmark/cmd/ci/main.go

```diff
@@ -32,7 +32,7 @@ type BenchmarkInfo struct {
 const (
 	loadType                    = "token-transfer"
 	metricport                  = uint(8080)
-	accessNodeAddress           = "127.0.0.1:3569"
+	accessNodeAddress           = "127.0.0.1:4001"
```

Comment:
Would this be safer and more maintainable to cast port `4001` as a global constant in the codebase?

Reply:
There's a new `ports.nodes.json` file that's generated during bootstrap which could be consumed here to get the actual port configured. This would be the preferred way to do it now. That requires a bit more work that I didn't want to include here. Just needed to fix the loader for testing.